hiss/user/admin.py

```diff
@@ -1,10 +1,44 @@
 from django.contrib import admin
+from django.utils import timezone
+
 from user.models import User
 
 
-# Register your models here.
+def check_in(modeladmin, request, queryset):  #TO-DO: Needs to be tested!!!
+    queryset.update(checked_in=True)
+    queryset.update(checked_in_datetime=timezone.datetime.now())
+
 class UserAdmin(admin.ModelAdmin):
-    pass
+    list_display = ("email", "is_active", "is_staff", "checked_in")
+    readonly_fields = ("email", "password", "is_active")
```

Comment:
```suggestion
    readonly_fields = ("email", "password")
```
We should allow admins to activate users on the off chance they don't get an email (this happened during HH)